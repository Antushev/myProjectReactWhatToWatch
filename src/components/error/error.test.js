import React from 'react';
import renderer from 'react-test-renderer';

import Error from './error.jsx';

describe(`ErrorComponent`, () => {
  it(`ErrorComponentSnapshot`, () => {
    const tree = renderer
      .create(
          <Error />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
