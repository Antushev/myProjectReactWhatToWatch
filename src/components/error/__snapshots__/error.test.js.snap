// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorComponent ErrorComponentSnapshot 1`] = `
<div
  className="page-content"
>
  <section
    className="catalog"
  >
    <div
      className="catalog__movies-list"
    >
      <h1>
        Кажется, сервер недоступен...
      </h1>
    </div>
  </section>
</div>
`;
